### ISSUE TITLE
design ShuffleNetV2 search space

### ISSUE THREAD

**Issue by pawopawo** [zhanghang1989/RegNet-Search-PyTorch#8]
Thanks for your excellent work!
Are the search criteria in regnet suitable for ShuffleNetV2? If I want to search a shuffleNet model with FLOPs 60M, can I design a search space based on this code?

**Comment by zhanghang1989** [comment]
Yes, the code is easy to extend. You can create another file for shuffleNet the the folder https://github.com/zhanghang1989/RegNet-Search-PyTorch/tree/master/arch

**Comment by pawopawo** [comment]
Thank you for your reply,

Do different blocks (shufflenet or ResNeSt) and different FLOPs require manual adjustment of the range of these [hyper-parameters](https://github.com/zhanghang1989/RegNet-Search-PyTorch/blob/10bb5967eb5a34521f52c270a7c195f348295b25/arch/regnet.py#L133)?

**Comment by zhanghang1989** [comment]
The default ranges are quite large. It's better to adjust them according to the FLOPs to improve the sampling efficiency.
